services/web/server/src/simcore_service_webserver/login/errors.py

```diff
@@ -0,0 +1,35 @@
+import functools
+
+from aiohttp import web
+from servicelib.aiohttp.typing_extension import Handler
+from servicelib.mimetype_constants import MIMETYPE_APPLICATION_JSON
+
+from ..errors import WebServerBaseError
+from ._constants import MSG_2FA_UNAVAILABLE_OEC
+
+
+class LoginError(WebServerBaseError, ValueError):
+    ...
+
+
+class SendingVerificationSmsError(LoginError):
+    msg_template = "Sending verification sms failed. {reason}"
+
+
+class SendingVerificationEmailError(LoginError):
+    msg_template = "Sending verification email failed. {reason}"
```

Comment:
As discussed. I do not think having a message here brings anything since
1) the handler below transform to an http error with a different message
2) the name of the class reflects the problem

Reply:
Yes, because of inheritance, I had trouble removing the message template, so I will leave it there.

Reply:
Why then you have `reason`? What kind of troubles?

Reply:
Yes, you are right, reason is not necessary there :+1: I will be moving on now and closing this PR.